### ISSUE TITLE
No error for incorred XML

### ISSUE THREAD

**Issue by hpatoio** [apigee/apigeelint#357]
I've this XML. Please note the double `</Request>`

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ProxyEndpoint name="default">
    <PreFlow name="PreFlow">
        <Request>
            <Step>
                <Name>oas-validation</Name>
            </Step>
        </Request>
        </Request>
        <Response/>
    </PreFlow>
    <Flows/>
    <PostFlow name="PostFlow">
        <Request/>
        <Response/>
    </PostFlow>
    <HTTPProxyConnection>
        <BasePath>/foo-1</BasePath>
        <VirtualHost>fpp-app</VirtualHost>
    </HTTPProxyConnection>
    <RouteRule name="default">
        <TargetEndpoint>default</TargetEndpoint>
    </RouteRule>
</ProxyEndpoint>
```

No errors or warning are reported by `apigeelint` but when I run `apigeetool` (https://github.com/apigee/apigeetool-node) I get this error

```
Deployment result: "JAXBException occurred : Unexpected close tag </Request>; expected </PreFlow>.\n at [row,col {unknown-source}]: [12,17]. Unexpected close tag </Request>; expected </PreFlow>.\n at [row,col {unknown-source}]: [12,17]. "
```

Is there a way to report error/warning when the XML is not correct ?

**Comment by DinoChiesa** [comment]
This is now fixed via #389.   
Thanks for reporting it.
